### ISSUE TITLE
missing imports: calculate_semivariance

### ISSUE THREAD

**Issue by jiangyuanshuang** [szymon-datalions/pyinterpolate#4]
![image](https://user-images.githubusercontent.com/50265935/59996727-bacc0a00-968d-11e9-9938-1bc4713b2e89.png)
in pyinterpolate/pyinterpolate-master/tutorials/notebooks/simple_kriging.ipynb， i find 
     from pyinterpolate.kriging.semivariance import calculate_semivariance  ,semivariance is no exist,What should I do? thanks you

**Comment by szymon-datalions** [comment]
Hello!

I've checked it and you were right. To this point it was impossible to run these commands. I've updated project and now everything should work fine. I removed **calculate_semivariance** method when I had refactored code for the areal semivariance calculations.

I've restored this method but in the final version of the library it will be removed and enclosed in the Semivariance class. Actually, there is no need to invoke warning every time when it will be used. Information is given in the method's doctoring.

Thank you for the issue posting. Updated notebook is available here: https://github.com/szymon-datalions/pyinterpolate/blob/master/tutorials/notebooks/simple_kriging.ipynb but you must reinstall library to run it.
